app/assets/stylesheets/users/_users.scss

```diff
@@ -0,0 +1,40 @@
+/* app/assets/stylesheets/users.scss */
+
+.profile-card {
+  display: flex;
+  align-items: center;
+  margin-bottom: 8px;
+  background: white;
+  padding: 40px 58px;
+  margin: 80px 200px 80px 200px;
```

Comment:
In the _users.scss file, the .profile-card and .profile-lesson styles have redundant margin properties. You may want to remove one set of margins to avoid conflicting styles.
.profile-card {
  margin: 80px 200px 80px 200px; 
}

.profile-lesson {
  margin: 80px 200px 80px 200px; 
}

Reply:
I am okay to adjust any of the margin elements as they are only temporary for us to see